python/lsst/cp/verify/mergeResults.py

```diff
@@ -439,3 +440,139 @@ class CpVerifyVisitRunMergeTask(CpVerifyRunMergeTask):
     _DefaultName = 'cpVerifyVisitRunMerge'
 
     pass
+
+
+class CpVerifyCalibMergeConnections(pipeBase.PipelineTaskConnections,
+                                    dimensions={"instrument"},
+                                    defaultTemplates={}):
+    inputStats = cT.Input(
+        name="exposureStats",
+        doc="Input statistics to merge.",
+        storageClass="StructuredDataDict",
+        dimensions=["instrument", "detector"],
+        multiple=True,
+    )
+
+    outputStats = cT.Output(
+        name="exposureStats",
+        doc="Output statistics.",
+        storageClass="StructuredDataDict",
+        dimensions=["instrument"],
+    )
+
+
+class CpVerifyCalibMergeConfig(pipeBase.PipelineTaskConfig,
+                               pipelineConnections=CpVerifyCalibMergeConnections):
+    """Configuration paramters for exposure stats merging.
+    """
+    runStatKeywords = pexConfig.DictField(
+        keytype=str,
+        itemtype=str,
+        doc="Dictionary of statistics to run on the set of exposure values. The key should be the test "
+        "name to record in the output, and the value should be the `lsst.afw.math` statistic name string.",
+        default={},
+    )
+
+
+class CpVerifyCalibMergeTask(pipeBase.PipelineTask, pipeBase.CmdLineTask):
+    """Merge statistics from detectors together.
+    """
+    ConfigClass = CpVerifyCalibMergeConfig
+    _DefaultName = 'cpVerifyCalibMerge'
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        inputs = butlerQC.get(inputRefs)
+
+        dimensions = [exp.dataId.byName() for exp in inputRefs.inputStats]
+        inputs['inputDims'] = dimensions
+
+        outputs = self.run(**inputs)
+        butlerQC.put(outputs, outputRefs)
+
+    def run(self, inputStats, inputDims):
+        """Merge statistics.
+
+        Parameters
+        ----------
+        inputStats : `list` [`dict`]
+            Measured statistics for a detector.
+        inputDims : `list` [`dict`]
+            List of dictionaries of input data dimensions/values.
+            Each list entry should contain:
+
+            ``"detector"``
+                detector id value (`int`)
+
+        Returns
+        -------
+        outputStats : `dict`
+            Merged full exposure statistics.
+
+        Notes
+        -----
+        The outputStats should have a yaml representation as follows.
+
+        VERIFY:
+          DetectorId1:
+            VERIFY_MEAN: boolean
+            VERIFY_SIGMA: boolean
+          DetectorId2:
+            [...]
+          MEAN_UNIMODAL: boolean
```

Comment:
Where do these UNIMODAL values come from?

Reply:
That was me confusing myself by copying a docstring and incompletely editing it.  The new docstring now represents what is actually constructed, and use generic `TEST_VALUE` style names.